Localization/localization.cfg

```diff
@@ -131,135 +131,135 @@ Localization
+        precisemaneuver_keybindings_proginc             = Incremenar progrado
+        precisemaneuver_keybindings_progdec             = Reducir progrado
+        precisemaneuver_keybindings_progzero            = Anular progrado
+        precisemaneuver_keybindings_norminc             = Incrementar normal
+        precisemaneuver_keybindings_normdec             = Reducir normal
+        precisemaneuver_keybindings_normzero            = Anular normal
+        precisemaneuver_keybindings_radinc              = Incrementar radial
+        precisemaneuver_keybindings_raddec              = Reducir radial
+        precisemaneuver_keybindings_radzero             = Anular radial
+        precisemaneuver_keybindings_timeinc             = Incrementar tiempo
+        precisemaneuver_keybindings_timedex             = Reducir tiempo
+        precisemaneuver_keybindings_circularize         = Circularizar órbita
+        precisemaneuver_keybindings_turnup              = Agregar una órbita
+        precisemaneuver_keybindings_turndown            = Reducir una órbita
+        precisemaneuver_keybindings_pageinc             = Cambiar incremento (w/Alt)
+        precisemaneuver_keybindings_pageconics          = Cambiar modo de órbita (w/Alt)
+        precisemaneuver_keybindings_encfocus            = Enfocar siguiente encuentro
+        precisemaneuver_keybindings_patchesmore         = Mostrar más parches
+        precisemaneuver_keybindings_patchesless         = Mostrar menos parches
+        precisemaneuver_keybindings_timex10             = Cambiar x10 veces
+        precisemaneuver_keybindings_nextnode            = Siguiente nodo de maniobra
+        precisemaneuver_keybindings_prevnode            = Anterior nodo de maniobra
+        precisemaneuver_keybindings_delnode             = Eliminar nodo de maniobra
+                
+        precisemaneuver_date_format                     = Año <<1>>, Día <<2>>, <<3>>:<<4>>:<<5>>
+        precisemaneuver_date_format_compact             = A<<1>> D<<2>> <<3>>:<<4>>:<<5>>
+                
+        precisemaneuver_node                            = Nodo
                 
         precisemaneuver_prefix_kilo                     = k
         precisemaneuver_prefix_mega                     = M
         precisemaneuver_prefix_giga                     = G
         precisemaneuver_meter                           = m
                 
-        precisemaneuver_pager_focus                     = Focus
-        precisemaneuver_pager_focus_tooltip             = Focus view on the maneuver node
-        precisemaneuver_pager_delete                    = Del
-        precisemaneuver_pager_delete_tooltip            = Delete current maneuver node
+        precisemaneuver_pager_focus                     = Enfocar
+        precisemaneuver_pager_focus_tooltip             = Enfocar vistar en nodo de maniobra
+        precisemaneuver_pager_delete                    = Eliminar
```

Comment:
The best here are "Eliminar" you could use "Elim". Is way difficult to find a 3 letters abbreviation here that is understandable. Probably "Supr" is the best way "Suprimir" is also a good translate  for "Delete"